### DISCUSSION TITLE
Any updates?

### DISCUSSION THREAD

**Discussion by neoOpus** [sienori/Tab-Session-Manager#1153] [General]
It looks like the project went into hibernation mode, meaning that development stopped on it. This unfortunately means that any existing bug fixes and feature updates won't be released. It also looks like the tab management extension isn't getting updated anymore.

**Comment by michaelarlt** [comment]
One month later, issues are stacking and PR's as well. No updates still. Project seems abandoned. Hopefully someone will fork it and continue

**Comment by brian163** [comment]
Has anyone hard from the lead developer @sienori on this project since Sept. 2023? Having used this add-on in Firefox for a long time as basically a session "insurance plan" running in the background, I just started using it's session tracking. And while quirky, really is pretty close to a perfect solution when you need a tab management solution with multi-computer syncing. I had just given it a positive review in the FF add-ons site and sent a donation. But in the hours since, having perused the project here, I have become more and more concerned this is could be heading towards abandonment. I really hope not but would love to hear from the dev along with his plans. (I suspect others would as well. And while technical I'm not a code developer so I can't offer to contribute. I appreciate those who I have seen do so.)

I understand it's a personal choice for an independent developer to create and maintain an add-on like this. But I also think in all fairness developers should stop soliciting for donations if their intentions/plans are to stop further development. (Not doing so just sows distrust for the independent dev community in general, IMO.)

Anyhow, just my 2 cents. And I hope this has just been a "break" (pause) for the dev. I do still appreciate what he created here for as long as it continues to function with ongoing Firefox platform updates.

**Comment by bjlockie** [comment]
> I also think in all fairness developers should stop soliciting for donations if their intentions/plans are to stop further development. (Not doing so just sows distrust for the independent dev community in general, IMO.)

It may not have been intentional to suddenly leave.

**Reply by neoOpus** [reply]
> > I also think in all fairness developers should stop soliciting for donations if their intentions/plans are to stop further development. (Not doing so just sows distrust for the independent dev community in general, IMO.)
> 
> It may not have been intentional to suddenly leave.

I should have been more thoughtful before voicing my complaints, and I genuinely regret my actions now. Thank you for the reminder, and I hope that @sienori is in a better shape than me and that everything is alright with his life.

**Comment by NamitaBi** [comment]
Wondering same as @sienori hasn't been active for a while.

**Comment by bohall00** [comment]
I was shocked to see an update for the Firefox extension today! This is a great addition and will be helpful to many. I don't dare to dream, but it would be very cool if @sienori has found renewed interest. So much has broken after falling years behind in development, but lots of us have been hanging on because if everything were to function as intended TSM is a better solution than any other extensions that I'm aware of.
